ru.max.MAX.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright MAX © 2025 -->
+<component type="desktop">
+  <id>ru.max.MAX</id>
+  <launchable type="desktop-id">ru.max.MAX.desktop</launchable>
+  <name>MAX</name>
+  <developer id="ru.max">
+    <name>max.ru</name>
+  </developer>
+  <summary>Fast and lightweight app for communication and everyday tasks</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://legal.max.ru/ps</project_license>
+  <url type="homepage">https://max.ru/</url>
+  <url type="help">https://help.max.ru/</url>
+  <content_rating type="oars-1.1"/>
+  <update_contact>help@max.ru</update_contact>
+  <description>
+    <p>Messenger with a built-in mini-app platform, chatbot builders, and a payment system.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://max.ru/s/img/phone-body.png</image>
```

Comment:
this needs a better screenshot than a phone frame


<img width="1118" height="495" alt="Image" src="https://github.com/user-attachments/assets/79c9fd32-1183-495a-8339-2b267e77cdb2" />